src/main/java/seedu/duke/commands/IncorrectCommand.java

```diff
@@ -5,15 +5,12 @@
  */
 public class IncorrectCommand extends Command {
 
-    public final String feedbackToUser;
-
     public IncorrectCommand(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
     }
 
     @Override
-    public CommandResult execute() {
-        return new CommandResult(feedbackToUser);
+    public void execute() {
```

Comment:
if this execute() method is empty, maybe we could consider getting rid of it? unless it serves a purpose :D

Reply:
Ok, I tried to remove the execute() method, but it threw an exception cause the Command class requires every child class to implement the execute class, so I'll just leave it there

Reply:
okie noted :D